### PR TITLE
feat: get cli args

### PR DESCRIPTION
**Pull request by load1n9**
Continuation of #22

### CODE REVIEW

File: runtime/src/ext/process.rs [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    }
+
+    fn internal_get_cli_args(agent: &mut Agent, _this: Value, _: ArgumentsList) -> JsResult<Value> {
+        let args = env::args().skip(1).collect::<Vec<String>>();
+        let args = args.iter().map(|s| s.as_str()).collect::<Vec<&str>>();
+        let args = args
+            .iter()
+            .map(|s| nova_vm::ecmascript::types::String::from_string(agent, s.to_string()))
+            .collect::<Vec<_>>();
+        // TODO: This should be an array
```
Comment (lino-levan) [inline runtime/src/ext/process.rs:35]: What, please elaborate

File: runtime/src/ext/process.rs [hunk 2]
```diff
@@ -0,0 +1,39 @@
+use andromeda_core::{Extension, ExtensionOp};
+use nova_vm::ecmascript::{
+    builtins::{ArgumentsList, Array},
+    execution::{Agent, JsResult},
+    types::Value,
+};
+use std::env;
+
+/// Process extension for Nova
```
Comment (marc2332) [inline runtime/src/ext/process.rs:9]: Nova ? :trollface: 
```suggestion
/// Process extension for Nova
```

File: runtime/src/ext/process.rs (hunk 1 above)
Comment (marc2332) [reply runtime/src/ext/process.rs:35]: +1

**Review by marc2332 — CHANGES_REQUESTED** [review]

File: runtime/src/ext/process.rs (hunk 1 above)
Comment (load1n9) [reply runtime/src/ext/process.rs:35]: oh sorry left that there

**Review by lino-levan — APPROVED** [review]
Don't love that we're basing our nova off of a branch but /shrug

**Pull request merged**